node/windows-packaging/CalicoWindows/kubernetes/kubelet-service.ps1

```diff
@@ -58,6 +65,7 @@ $argList = @(`
 if (Get-IsContainerdRunning)
 {
     Write-Host "Detected containerd running, configuring kubelet for containerd"
+    $argList += "--container-runtime=remote"
```

Comment:
This parameter seems deprecated in latest version.

Reply:
ok